CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ This project adheres to the American Express [Code of Conduct](./CODE_OF_CONDUCT
 3. Install the dependencies by running
 
     ```bash
-    $ yarn
+    $ yarn@^1
```

Comment:
wait, this was supposed to be the install command 🤦

Reply:
Maybe just add a line saying to use yarn@1?  Reinstalling it globally might not be appreciated.

Reply:
changed to `$ npx yarn@^1` in 89e092c